lib/utilities/get-script-file-name.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function getScriptFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
+  return Array.isArray(filePath.match(scriptFileNameRegEx)) ? filePath.match(scriptFileNameRegEx)[0] : '';
```

Comment:
Can you explain why we need this regexp matching?

Also, we need to simplify this:

```suggestion
  let match = filePath.match(scriptFileNameRegEx);
  return match ? match[0] : '';
```

Reply:
That Makes sense. I updated the code.